helper-cli/src/main/kotlin/commands/packageconfig/CreateCommand.kt

```diff
@@ -113,25 +113,17 @@ internal class CreateCommand : CliktCommand(
     }
 
     private fun createPackageConfiguration(provenance: Provenance): PackageConfiguration =
-        when (provenance) {
-            is ArtifactProvenance -> {
-                PackageConfiguration(
-                    id = packageId,
-                    sourceArtifactUrl = provenance.sourceArtifact.url
+        PackageConfiguration(
+            id = packageId,
+            sourceArtifactUrl = provenance.let { it as? ArtifactProvenance }?.let {
```

Comment:
Simplify to: `(provenance as? ArtifactProvenance)?.sourceArtifact?.url`

Reply:
It seems while you reviewed I merged already. I've addressed this with https://github.com/oss-review-toolkit/ort/pull/5687.

Reply:
The code in that PR is still more complicated than my proposal here.

Reply:
Oh sorry, it was too late yesterday. I'll fix it up.

Reply:
Hope this is it now: https://github.com/oss-review-toolkit/ort/pull/5689